redwood-treehouse/src/commonMain/kotlin/app/cash/redwood/treehouse/StateSnapshot.kt

```diff
@@ -33,6 +34,10 @@ public class StateSnapshot(
       entry.value.map { mutableStateOf(it.fromJsonElement()) }
     }
   }
+
+  @JvmInline
+  @Serializable
+  public value class Id(public val value: String?)
```

Comment:
TIL `value` classes can be nested; I always thought they had to be top level.

Reply:
That was indeed a limitation a few years ago but it has since been lifted.